openshift/cirras-underwriting-jasper-deployment.yaml

```diff
@@ -99,6 +99,14 @@ spec:
                 configMapKeyRef:
                   name: cirras-underwriting-jasper-config-#{ENV}#
                   key: JASPERREPORTS_UI_PASSWORD
+          livenessProbe:
```

Comment:
This isn't strictly necessary anymore; the Jasper deployments are being permanently scaled to 0 as part of the 3.5.0 release. Not sure if that would cause this liveness probe to run?

Reply:
The liveness probe will only run if there are deployments running